### ISSUE TITLE
apps/v1beta{1,2,} deprecated under k8s 1.16

### ISSUE THREAD

**Issue by drewboswell** [bitnami-labs/sealed-secrets#278]
The references to apps/v1beta 1,2,none need to be replaced with "apps/v1". Otherwise this will no longer work under kubernetes 1.16.

**Comment by mkmik** [comment]
Uh, wasn't this solved in #269?

Are you using the helm chart?

**Comment by jjo** [comment]
> Uh, wasn't this solved in #269?

kube-libsonnet already has the fixes on master, looks like sealed-secrets need syncing its `jb` manifest for those ?

**Comment by mkmik** [comment]
@jjo: it was manually overridden in #272. I will cleanup up now that upstream is fixed; I didn't want to ulput unnecessary pressure but I needed to cut v0.9.0.

We really need to do something about the helm charts; I'm never sure when I see a bug report like this whether the users are using the latest version.

Perhaps we should setup a "GitHub ssue template"

**Comment by jjo** [comment]
> @jjo: it was manually overridden in #272. I will cleanup up now that upstream is fixed; I didn't want to ulput unnecessary pressure but I needed to cut v0.9.0.
> 
ah cool, appreciate if you do so, as our internal `sre-kube-manifests` are stuck to `sealed-secrets` «commanding» re: `jb` :P ->
From https://github.com/jsonnet-bundler/jsonnet-bundler#current-limitations:
```
* If two dependencies depend on the same package (diamond problem), they must require the same version
```
i.e. as it's vendor/ed there, it does mandate our repo kube-libsonnet commit.

**Comment by drewboswell** [comment]
I am using the released files on GitHub, which still have references to the deprecated APIs.

**Comment by mkmik** [comment]
Yeah, we didn't yet cut a release with that fix.
Will release ASAP, thanks for the reminder.

Closing as duplicate of #269

**Comment by mkmik** [comment]
Fix released in [v0.9.1](https://github.com/bitnami-labs/sealed-secrets/releases/tag/v0.9.1)
